src/sidebar/index.js

```diff
@@ -1,11 +1,44 @@
-import { parseJsonConfig } from '../boot/parse-json-config';
+// Load polyfill for :focus-visible pseudo-class.
+import 'focus-visible';
+// The entry point component for the app.
```

Comment:
Can we configure the plugin not to remove the new blank lines above commented imports?

Reply:
Let me check if this can be done.